### PR TITLE
[Refactor][doc]Edit the ecological extension part of the document

### PR DESCRIPTION
**Pull request by caoliang-web**
## Problem Summary:

1.Edit the ecological extension part of the document

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: new-docs/zh-CN/ecosystem/external-table/hive-bitmap-udf.md [hunk 1]
```diff
@@ -24,4 +24,81 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Hive UDF
```
Comment (hf200012) [inline new-docs/zh-CN/ecosystem/external-table/hive-bitmap-udf.md:27]: Delete this and put it in spark load

**Review by hf200012 — COMMENTED** [review]
Improve English documents

File: new-docs/zh-CN/ecosystem/external-table/hive-bitmap-udf.md (hunk 1 above)
Comment (caoliang-web) [reply new-docs/zh-CN/ecosystem/external-table/hive-bitmap-udf.md:27]: ok,I modified it

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**